modules/__tests__/StaticRouter-test.js

```diff
@@ -186,6 +186,30 @@ describe('StaticRouter', () => {
         })
       })
 
+      it('coerces query values to strings', () => {
+        assertParsedDescriptor({
+          query: { a: 23, b: false }
+        }, {
+          pathname: '',
+          query: { a: '23', b: 'false' },
+          hash: '',
+          state: null,
+          search: '?a=23&b=false'
+        })
+      })
+
+      it('does not try to coerce null or undefined to strings', () => {
+        assertParsedDescriptor({
+          query: { a: undefined, b: null }
+        }, {
+          pathname: '',
+          query: { a: undefined, b: null },
```

Comment:
Should it even let `undefined` through here? If we parse `?b`, the query won't have an `a` key.